packages/client/src/index.ts

```diff
@@ -5,6 +5,7 @@ export const PenumbraSymbol = Symbol.for('penumbra');
 export interface PenumbraInjection {
   readonly connect: () => Promise<MessagePort>;
   readonly request: () => Promise<void>;
+  readonly disconnect: () => Promise<void>;
   readonly isConnected: () => boolean | undefined;
```

Comment:
this is the same client api we had before.

undefined means that a connection state has not been resolved - it is unattempted or pending. after attempting to connect, or after closing a connection, it may be 'false'.

a caller can understand that a truthy response indicates the connection is active, and a falsy value indicates the connection is not active.